kbase-extension/static/kbase/js/common/jobManager.js

```diff
@@ -255,13 +257,18 @@ define([
         /**
          * Add a bus listener for ${event} messages
          *
-         * @param {string} type - a valid message type to listen for (see validOutgoingMessageTypes)
+         * @param {string} msgType - a valid message type to listen for (see validOutgoingMessageTypes)
+         * @param {string} channelType - the channel type to listen to (one of the values in jcm.CHANNEL)
```

Comment:
add in an argument to specify the type of channel to listen to instead of using `JOB` by default